xml/deployment_scenarios.xml

```diff
@@ -0,0 +1,1175 @@
+   <title>Container Registry Mirror</title>
+   <note>
+    <title>Mirroring Multiple Image Registries / Chart Repositories</title>
+    <para>
+     You can mirror images and charts from multiple registries in one shared
+     internal registry. We do not recommend mirroring multiple registries in a
+     shared registry due to the potential conflicts.
+    </para>
+    <para>
+     We highly recommend running separate helm chart and container registry
+     mirrors for each source registry.
+    </para>
+    <para>
+     Additional mirror registries must be run on separate mirror servers for
+     technical reasons.
+    </para>
+   </note>
+
+   <sect3 xml:id="sec.deploy.scenarios.airgap.container-registry.mirror">
+    <title>Mirror Configuration</title>
+    <para>
+     The container image registry is provided as a container image itself.
+     You must download the registry container from &suse; and run it on the
+     respective server.
+    </para>
+    <note>
+     <title>Internal Registry Mirror Is Read Only</title>
+     <para>
+      For security reasons, the internal registry mirror is configured in
+      <literal>read-only</literal> mode. Therefore, pushing container images to
+      this mirror will not be possible. It can only serve images that were
+      previously pulled and cached by the external mirror and then uploaded to
+      the internal mirror.
+     </para>
+     <para>
+      You can modify and store your own container images on the external
+      registry and transfer them with the other container images using the same
+      process. If you need to be able to modify and store container images on
+      the internal network, we recommend to create a new registry that will hold
+      these images. The steps needed to run your own full container image
+      registry are not part of this document.
+     </para>
+      <para>
+      For more information you can refer to:
+      <link xlink:href="https://www.suse.com/documentation/sles-15/book_sles_docker/data/sec_docker_registry_definition.html">SLES15 - Docker Open Source Engine Guide: What is Docker Registry?</link>.
+     </para>
+    </note>
+
+    <procedure>
+     <title>Set Up The External Mirror</title>
+     <step>
+      <para>
+       SSH into the external mirror server.
+      </para>
+     </step>
+     <step>
+      <para>
+       Install <filename>docker</filename>, <filename>helm-mirror</filename>
+       and <filename>skopeo</filename>.
+      </para>
+<screen>&prompt.root;zypper in docker helm-mirror skopeo
+      </screen>
+     </step>
+     <step>
+      <para>
+       Start the docker service and enable it at boot time:
+      </para>
+<screen>&prompt.root;<command>systemctl enable --now docker.service</command>
+         </screen>
+     </step>
+     <step>
+      <para>
+       Pull the registry container image from &suse;.
+      </para>
+<screen>&prompt.root;<command>docker pull registry.suse.com/sles12/registry:2.6.2</command>
```

Comment:
I cannot pull this image, will it be published later on?

```
sascha@sgrunert-test:~> docker pull registry.suse.com/sles12/registry:2.6.2
Error response from daemon: manifest for registry.suse.com/sles12/registry:2.6.2 not found
```

Reply:
It is now